bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/select.js

```diff
@@ -192,7 +192,20 @@ pimcore.object.classes.data.select = Class.create(pimcore.object.classes.data.da
             autoHeight: true,
             plugins: [
                 Ext.create('Ext.grid.plugin.CellEditing', {
-                    clicksToEdit: 1
+                    clicksToEdit: 1,
+                    listeners: {
+                        edit: function(editor, e) {
+                            if(!e.record.get('value')) {
+                                e.record.set('value', e.record.get('key'));
+                            }
+                        },
+                        beforeedit: function(editor, e) {
+                            if(e.field === 'value') {
```

Comment:
just one last note: would it make sense to also avoid duplicates?

Reply:
just added this. when you set a value to an already existing value, it gets reverted (silently). Do you think we need a notification message?

Reply:
Guess we can still add it later if it feels strange.